src/components/Slider/Slider.js

```diff
@@ -104,10 +104,12 @@ export default class Slider extends PureComponent {
 
   static defaultProps = {
     hideTextInput: false,
+    step: 1,
     stepMuliplier: 4,
     disabled: false,
     minLabel: '',
     maxLabel: '',
+    onChange: () => {},
```

Comment:
Do we need this default prop even though the code does `props.onChange` type check?

Reply:
Good question! The reason I added the type check was in case someone had passed an incorrect prop type: if someone passes in a string for instance, _instead of a fatal error_, it would simply do nothing (except warn them in the console that the PropType was wrong). If you think this is unnecessary or too much complexity, I'll remove it. Was going back and forth on that one myself.

Reply:
Thank you for the thought @i-a-n! The type-check is definitely a good thing 👍 - And I think the type check you added makes the default prop here no longer needed. Thanks!